src/test/groovy/org/gradle/profiler/JFRProfilerIntegrationTest.groovy

```diff
@@ -138,32 +139,44 @@ class JFRProfilerIntegrationTest extends AbstractProfilerIntegrationTest {
         latestSupportedGradleVersion  | _
     }
 
+    @Requires({ !OperatingSystem.isWindows() })
+    // No perl installed on Windows
     @Unroll
-    def "can profile Gradle #versionUnderTest using JFR, `gradle` command and no daemon"() {
+    def "can profile Gradle no daemon #versionUnderTest with #iterations iterations"(String versionUnderTest, int iterations) {
         given:
         instrumentedBuildScript()
 
         when:
-        new Main().run("--project-dir", projectDir.absolutePath, "--output-dir", outputDir.absolutePath, "--gradle-version", versionUnderTest, "--profile", "jfr", "--no-daemon", "assemble")
+        new Main().run(
+            "--project-dir", projectDir.absolutePath,
```

Comment:
It would be nice to add a cleanup action here to see that this is supported.

Reply:
I created a new test.